app/views/static_pages/faq/_faq_accordion.html.erb

```diff
@@ -1,5 +1,223 @@
-<div id="faq-accordion" role="tablist" aria-multiselectable="true">
-  <% faq_items.each_with_index do |faq_item, index| %>
-    <%= render 'static_pages/faq/faq_card', faq_item: faq_item, index: index %>
-  <% end %>
-</div> 
+<div class="max-w-7xl mx-auto py-12 px-4 sm:py-16 sm:px-6 lg:px-8">
+  <div class="max-w-3xl mx-auto divide-y-2 divide-gray-200">
+    <dl class="space-y-6 divide-y divide-gray-200">
+      <div class="pt-6" data-controller="reveal" data-reveal-toggle-classes-value='["-rotate-180", "rotate-0"]'>
```

Comment:
Thanks for inlining the faq items, I think it's definitely the way to go until we land on how/where we want to store them for good. 

Seeing this all together now, I think you're right about using view components to reduce some of the duplication. I think we could get away with just having a faq item component instead of an overarching accordion component. The items would still essentially be inline, but their similarities would be abstracted away. 

Something like this using slots perhaps?
```html
<%= render FaqItemComponent.new do |item| %>
  <% item.question("Who is The Odin Project for?") %>
  <% item.answer do %>
    <p class="text-gray-500">The Odin Project is for absolute beginners in the world of web development and those who have tried other resources without success.  In reality, The Odin Project is for anyone who wants to work as a web developer, do freelancing work, build a startup, or just hack together a homepage for yourself.  All the tools you need are here. <strong>You should consider yourself fairly competent with personal computers and open to using a non-Windows based Operating System.</strong></p>
    <p class="text-gray-500">We’ve built an in-depth curriculum designed to take you 100% of the way to becoming a web developer. Many other resources out there teach you just the basics of programming.  With The Odin Project you’ll create your own websites, games, and social media sites.</p>
    <p class="text-gray-500">The curriculum has also been designed in a modular fashion, making skimming through lessons a breeze.</p>
  <% end %>
<% end %>

<%= render FaqItemComponent.new do |item| %>
  <% item.question(" What is the goal of The Odin Project?") %>
  <% item.answer do %>
    <p class="text-gray-500">The goal of The Odin Project is to provide a complete path for students to go from zero programming knowledge to employed as web developers while working with other students and picking up the skills to become self sufficient along the way.</p>
  <% end %>
<% end %>
```